### PR TITLE
#CU-NIX-31 subxt gen prove no need complex setup

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
## Checklist

- [ ] Updated the rust/typescript docs.
- [ ] Updated the `docs/`.

### CODE REVIEW

**Comment by dzmitry-lahoda** [comment]
@obsessed-cake would work on this to progress on client generation

File: scripts/dali-dev-scale-export.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+PARA_PORT=9188
```
Comment (dzmitry-lahoda) [inline scripts/dali-dev-scale-export.sh:3]: please use port we use in other places, please try to make scripts more consistent regarding defaults

File: scripts/dali-dev-scale-export.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+../../../paritytech/polkadot/target/release/polkadot build-spec --chain=rococo-local --raw > rococo-local-raw.json
+
+result/bin/polkadot --chain=./rococo-local-raw.json --tmp &
+polkadot_pid=$(echo $!)
+
+../target/release/composable --tmp --force-authoring  --chain=dali-dev  --discover-local --ws-port $PARA_PORT --  --chain=./rococo-local-raw.json &
+composable_pid=$(echo $!)
+
+# wait for TCP port opening
+while ! nc -z 127.0.0.1 $PARA_PORT; do
```
[the remark this answers is not in the document]
Comment (dzmitry-lahoda) [reply scripts/dali-dev-scale-export.sh:15]: This PR is on hold now (as work shifted to bridge again). This is draft bash to be refactored and moved into nix. So this SH as it is no is okey, we are not.  It just proves one no need for zombinet/polkalauncher/cluster to get export SCALE locally.

[the remark this answers is not in the document]
Comment (dzmitry-lahoda) [reply scripts/dali-dev-scale-export.sh:15]: `127.0.0.1` <- yeah, good refactoring. easier to be nixied later. 
so this scripts just proves how easy to run locally for client gen.

File: scripts/dali-dev-scale-export.sh (hunk 1 above)
Comment (obsessed-cake) [reply scripts/dali-dev-scale-export.sh]: fixed.

File: scripts/dali-dev-scale-export.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (obsessed-cake) [reply scripts/dali-dev-scale-export.sh]: fixed.

**Review by andyjsbell — APPROVED** [review]
lgtm 👍

**Pull request merged**